lib/components/profile_card.dart

```diff
@@ -61,43 +63,85 @@ class _ProfileCardState extends State<ProfileCard> {
   }
 
   Widget _loggedInView(BuildContext context) {
+    return ArDriveClickArea(
+      child: ScreenTypeLayout.builder(
+        mobile: (context) => _buildLoggedInViewForPlatform(
+          context,
+          isMobile: true,
+        ),
+        desktop: (context) => _buildLoggedInViewForPlatform(
+          context,
+          isMobile: false,
+        ),
+      ),
+    );
+  }
+
+  Widget _buildLoggedInViewForPlatform(
+    BuildContext context, {
+    required bool isMobile,
+  }) {
     final state = context.read<ProfileCubit>().state as ProfileLoggedIn;
     final walletAddress = state.walletAddress;
 
-    return ArDriveClickArea(
-      child: ArDriveOverlay(
-        onVisibleChange: (visible) {
-          if (!visible) {
-            setState(() {
-              _showProfileCard = false;
-            });
-          }
-        },
-        visible: _showProfileCard,
-        anchor: const Aligned(
-          follower: Alignment.topRight,
-          target: Alignment.bottomRight,
-          offset: Offset(0, 4),
-        ),
-        content: _buildProfileCardContent(context, state, walletAddress),
-        child: _buildProfileCardHeader(context, walletAddress),
+    return ArDriveOverlay(
+      onVisibleChange: (visible) {
+        if (!visible) {
+          setState(() {
+            _showProfileCard = false;
+          });
+        }
+      },
+      visible: _showProfileCard,
+      anchor: Aligned(
+        follower: Alignment.topRight,
+        target: Alignment.bottomRight,
+        offset: isMobile ? const Offset(12, -60) : const Offset(0, 4),
+      ),
+      content: _buildProfileCardContent(
+        context,
+        state,
+        walletAddress,
+        isMobile: isMobile,
       ),
+      child: _buildProfileCardHeader(context, walletAddress),
     );
   }
 
   Widget _buildProfileCardContent(
     BuildContext context,
     ProfileLoggedIn state,
-    String walletAddress,
-  ) {
+    String walletAddress, {
+    bool isMobile = false,
```

Comment:
I'd prefer to have it as required
```suggestion
    required bool isMobile,
```